src/octoprint/plugins/action_command_notification/templates/action_command_notification_settings.jinja2

```diff
@@ -16,4 +16,11 @@
             </label>
         </div>
     </div>
+    <div class="control-group">
+        <label class="control-label" for="settings-serialAdditionalPorts">{{ _('Notification Filters') }}</label>
+        <div class="controls">
+            <textarea rows="4" class="block" id="settings-serialAdditionalPorts" data-bind="value: settings.plugins.action_command_notification.filters"></textarea>
+            <span class="help-inline">{{ _("Use this to define filters to hide unwanted notifications. One regular expression per line. Notifications that match a regular expression will not be shown.") }}</span>
```

Comment:
Should probably get a link to the Python regex docs, as not everyone would know how to use this otherwise.